format/src/main/scala/org/scalafmt/internal/TreePrinter.scala

```diff
@@ -55,7 +55,7 @@ object TreePrinter {
             val (quoteStyle, dquote) = dQuote(t.value)
             val dvalue = dRaw(t.value, quoteStyle)
             dquote + dvalue + dquote
-          case _ => text(tree.syntax) // ???
+          case _ => text(tree.syntax)
```

Comment:
This line is still a TODO, it's the only place where we currently fallback to TreeSyntax.

Reply:
Ah ok. I will replace with all the literals.